src/data/cameraData.ts

```diff
@@ -0,0 +1,30 @@
+interface cameraDetails {
+  name: string;
+  code: string;
+}
+
+const fhaz: cameraDetails = {
+  name: "Front Hazard Avoidance",
+  code: "fhaz",
+};
+
+const rhaz: cameraDetails = {
+  name: "Rear Hazard Avoidance",
+  code: "rhaz",
+};
+
+const navcam: cameraDetails = {
+  name: "Navigation",
+  code: "navcam",
+};
+
+const mast: cameraDetails = {
+  name: "Mast",
+  code: "mast",
+};
+
+export const roverCameras = {
```

Comment:
You can set this type using `RoverName` from the below file:
```suggestion
export const roverCameras: { [name: RoverName]: CameraDetails[] } = {
```

Reply:
change not completed as agreed